src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -807,7 +807,14 @@ void SolverBase::SolveSystem( DofManager const & dofManager,
 //  ++count;
 
 
-  GEOSX_WARNING_IF( !m_linearSolverResult.success(), "Linear solution failed" );
+  if( params.stopIfError )
```

Comment:
I do wonder if this option should be a part of `NonlinearSolverParameters` since fundamentally it affects the nonlinear loop behavior, not so much the linear solve. We could think of a proper name and set of choices to control behavior with respect to both breakdowns and non-convergence. @castelletto1 @joshua-white @rrsettgast opinions?